driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1460,7 +1479,7 @@ else if (host.state == Host.State.DOWN)
                 configuration.getPolicies().getRetryPolicy().init(Cluster.this);
                 reconnectionPolicy().init(Cluster.this);
                 configuration.getPolicies().getAddressTranslator().init(Cluster.this);
-                for (LatencyTracker tracker : trackers)
```

Comment:
Shouldn't there also be something here for calling `onRegister` on `errorTrackers`?